catalog.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Дизайн-студия «nёrd's»</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body class="page-body">
+  <header class="main-header">
+    <nav class="main-navigation">
+      <a class="main-header-logo" href="index.html">
+        <img src="img/index-logo.svg" width="160" height="65" alt="Логотип дизайн-студии «nёrd's»">
+      </a>
+      <ul class="site-navigation">
+        <li><a class="site-navigation-link" href="blank.html">Студия</a></li>
+        <li><a class="site-navigation-link" href="blank.html">Клиенты</a></li>
+        <li><a class="site-navigation-link site-navigation-link-current" href="catalog.html">Магазин</a></li>
```

Comment:
Поскольку это активная ссылка, то `href` нужно убрать:
```suggestion
        <li><a class="site-navigation-link site-navigation-link-current">Магазин</a></li>
```